llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -129,39 +129,30 @@ def __init__(
             mlflow_logging_config = {}
         if self.mlflow_registered_model_name is not None:
             import numpy as np
-            from mlflow.models.signature import ModelSignature
-            from mlflow.types.schema import ColSpec, Schema
 
             # Both the metadata and the task are needed in order for mlflow
             # and databricks optimized model serving to work
-            default_metadata = {'task': 'llm/v1/completions'}
             passed_metadata = mlflow_logging_config.get('metadata', {})
-            mlflow_logging_config['metadata'] = {
-                **default_metadata,
-                **passed_metadata
-            }
-            mlflow_logging_config.setdefault('task', 'text-generation')
-
-            # Define a default input/output that is good for standard text generation LMs
-            input_schema = Schema([
-                ColSpec('string', 'prompt'),
-                ColSpec('double', 'temperature', optional=True),
-                ColSpec('integer', 'max_tokens', optional=True),
-                ColSpec('string', 'stop', optional=True),
-                ColSpec('integer', 'candidate_count', optional=True)
-            ])
-
-            output_schema = Schema([ColSpec('string', 'predictions')])
-
-            default_signature = ModelSignature(inputs=input_schema,
-                                               outputs=output_schema)
```

Comment:
why don't we need this anymore?

Reply:
they told me that when using `llm/v1/{completions|chat}`, the input schema and signature would be added automatically, so no need to hardcode here, which would be more brittle.